### ISSUE TITLE
💡 Proposal - Accessibility

### ISSUE THREAD

**Issue by denkristoffer** [contentful/forma-36#601]
# Forma 36 contribution proposal

## The problem

When I worked on the Dropdown component I noticed it lacks the necessary accessibility like keyboard navigation. After asking around a bit it sounds like there are probably other components that could also be improved. These enhancements would greatly benefit both Contentful products and external users of Forma 36.

## The proposed solution

1. Make a list of components with a need for accessibility improvements.
2. List the required improvements.
3. Implement them one component by one.

## Breaking changes

It should be possible to do this work without breaking changes, but I suspect it'll be made much easier with hooks, so collecting the changes for an eventual 4.0 might make more sense. What do you think?

**Comment by burakukula** [comment]
@denkristoffer great proposal! I think we might want to gather those accessibility related changes and roll out major version. It would be great to make a bigger announcement out of it, there are some clients and external users that has been asking about those improvements.
I guess the `Autocomplete` would need some improvements as well - @suevalov I think you mentioned it during the interview. Alex, maybe you would have other components in mind, that needs improvement?
If you need any help with auditing what we have right now and what improvements we need to do, let me know. Maybe @mshaaban0 has already some input regarding this topic?
It might be a bigger topic, maybe we could create epic for it?

**Comment by denkristoffer** [comment]
Note for self, this is a really good resource: https://www.w3.org/TR/wai-aria-practices/
